### DISCUSSION TITLE
automated prayer metronome

### DISCUSSION THREAD

**Discussion by osrsguy** [runelite/runelite#18154] [Ideas]
When prayer is activated metronome will activate for example for 5 seconds,THIS would help with 1tick flicking prayer and this would remove need to turn on/off or listen normal metronome for 24/7 when its not needed.

**Comment by cdfisher** [comment]
This would probably be better off as a hub plugin rather than something in core RuneLite.

**Reply by osrsguy** [reply]
u mean i should post this somewhere else? if so where? @cdfisher

**Reply by Felanbird** [reply]
> u mean i should post this somewhere else? if so where? @cdfisher

Here is fine, it's just a note for anyone who looks at this and wants to implement it. I'll add the relevant label
